plugin/evm/gossiper.go

```diff
@@ -554,7 +554,7 @@ func (h *GossipHandler) HandleEthTxs(nodeID ids.NodeID, msg message.EthTxsGossip
 			if err == txpool.ErrAlreadyKnown {
 				h.stats.IncEthTxsGossipReceivedKnown()
 			} else {
-				h.stats.IncAtomicGossipReceivedError()
```

Comment:
We can still use and increase this after `h.vm.mempool.AddTx(&tx); err != nil ` if we'd like.

Reply:
Yeah, good point.